src/current.js

```diff
@@ -0,0 +1,88 @@
+import artsyXapp from "@artsy/xapp"
```

Comment:
`current` is a somewhat confusing name. What does it mean? Current as to what? Perhaps a comment can be added to the top of the file.

Reply:
I agree, naming is hard. What should we call old force is a hard question. A suggestion would be helpful if you have any and would make for an actionable comment.

Reply:
There's a question here (what does `current` mean) as well as a suggestion (comment top of file).

Reply:
Current means whatever we want to call force, I mentioned that in my previous comment. I don't feel like a comment would add anything additional once the file has found a descriptive name.